lib/plugins/create/templates/azure-nodejs/README.md

```diff
@@ -1,133 +1,211 @@
-# Azure Functions - Quick Start
+# Azure Functions
```

Comment:
I'm not sure this README fits the template well (even though I see it follows the old version we already had in).

Maybe we can clean not relevant sections (as e.g. `Create a new Azure Function App`) ?

Template is used by `serverless create` command to create new projects. So newly created project documentation (if any) could be some short description of _further_ steps that developer may follow

Other templates don't come with a README, and if they, it looks more as: https://github.com/serverless/serverless/tree/7336bd47f3f7f85e6210c20aa200f4b948cfc333/lib/plugins/create/templates/openwhisk-nodejs

What could be good, is that we link here to Serverless Azure doc that contains all this valuable information (if I see correctly, it's mostly copy/paste here - which also makes further maintenance more problematic)

Reply:
I agree. While I see the value and appreciate the effort which was taken here I believe that this can be really hard to maintain and keep in sync with the docs. I'd propose to provide as little information as possible in this file and just reference / link to the docs.

Reply:
agree, will link this to `https://serverless.com/framework/docs/providers/azure/`